homeassistant/components/sharkiq/__init__.py

```diff
@@ -43,11 +50,19 @@ async def async_connect_or_timeout(ayla_api: AylaApi) -> bool:
 
 async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
     """Initialize the sharkiq platform via config entry."""
+    ayla_region = SHARKIQ_REGION_DEFAULT
+    if not hasattr(config_entry.data, CONF_REGION):
+        new = {**config_entry.data}
+        new[CONF_REGION] = ayla_region
+        hass.config_entries.async_update_entry(config_entry, data=new)
+    else:
+        ayla_region = config_entry.data[CONF_REGION]
```

Comment:
I think this can be reduced to:

```suggestion
    if CONF_REGION not in config_entry.data:
        hass.config_entries.async_update_entry(
            config_entry,
            data={**config_entry.data, CONF_REGION: SHARKIQ_REGION_DEFAULT}
        )
```